include/PR/sched.h

```diff
@@ -0,0 +1,38 @@
+#ifndef SCHED_H
+#define SCHED_H
+
+#include "PR/ultratypes.h"
+#include "ultra64/vi.h"
+#include "ultra64/sptask.h"
+
+#define OS_SC_NEEDS_RDP         0x0001
+#define OS_SC_NEEDS_RSP         0x0002
+#define OS_SC_DRAM_DLIST        0x0004
+#define OS_SC_PARALLEL_TASK     0x0010
+#define OS_SC_LAST_TASK         0x0020
+#define OS_SC_SWAPBUFFER        0x0040
+
+#define OS_SC_RCP_MASK          0x0003
+#define OS_SC_TYPE_MASK         0x0007
+
+#define OS_SC_DP                0x0001
+#define OS_SC_SP                0x0002
+#define OS_SC_YIELD             0x0010
+#define OS_SC_YIELDED           0x0020
+
+typedef struct OSScTask {
+    /* 0x00 */ struct OSScTask* next;
+    /* 0x04 */ u32      state;
+    /* 0x08 */ u32      flags;
+    /* 0x0C */ void*    framebuffer; // CfbInfo*
```

Comment:
Out of curiosity, why is this being changed to `void*`?

Reply:
This header is a libultra header,  so it should only contain general types provided by libultra.
`CfbInfo` is part of the Zelda64 specific scheduler (which uses parts of the libultra stuff).

Since libultra headers shouldn't be including Zelda64 specific headers, then it was just left as a `void*`

Reply:
That makes sense. With that in mind, I think I'd prefer to remove the `CfbInfo*` comment out of this header as well and remove all references to z64 specific stuff. Perhaps a macro would be better suited in the z64 scheduler header for this kind of documentation like 
`FRAMEBUFFER(task) ((CfbInfo*)(task)->framebuffer)`.

Reply:
That's actually a good idea. I'll change it